notes/returned-mail-received.vbs

```diff
@@ -166,7 +202,7 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
     '-------------------------------------------------------------------------------------------------DIALOG
```

Comment:
I believe the If statement here is unnecessary. The dialog handling will always require at least one of these to be 'Yes'